connect/mirror/src/test/java/org/apache/kafka/connect/mirror/DedicatedMirrorIntegrationTest.java

```diff
@@ -14,14 +14,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.connect.mirror.integration;
```

Comment:
Note for reviewers: this was required so that we can access isConfiguredAndRunning from the test without having to make the method public.

Reply:
This move isn't necessary; we can and should keep this test suite in the `integration` package.

It's fine to add `public` methods to the `MirrorMaker` class since that class is not part of the public API.